google/cloud/aiplatform/training_jobs.py

```diff
@@ -1015,6 +1097,20 @@ def __init__(
             model_serving_container_health_route
         )
 
+        self._model_serving_container_command = model_serving_container_command
```

Comment:
Can we take this opportunity to envelope all the Model arguments into an object. Perhaps we can create the instance of `gca_model.Model` here?

Reply:
I'll look into it

Reply:
The model needs `model_display_name`, which is passed in the `run` function and not the `init` function, so we can't create it here.

We COULD create the `gca_model.ModelContainerSpec` at `CustomTrainingJob.init` though. I'll go ahead and do that.

Reply:
Can't you create the model up front and set the `display_name` afterwards? ie.

```
self._managed_model = gca_model.Model(...)
```

```
self._managed_model.display_name = display_name
```

Reply:
I didn't know you could do that with protos. Let me try it.